src/tw-install.rc

```diff
@@ -0,0 +1,17 @@
+### tw-install.rc configuration file 
```

Comment:
Perhaps include a brief description of its contents, as the Google Style Guide for Bash recommends: https://google.github.io/styleguide/shellguide.html#file-header

Reply:
sure will look into it